seacatauth/authn/webauthn/handler.py

```diff
@@ -0,0 +1,166 @@
+import base64
+import logging
+
+import aiohttp.web
+import asab.web
+import asab.web.rest
+
+from ...decorators import access_control
+
+#
+
+L = logging.getLogger(__name__)
+
+#
+
+
+class WebAuthnHandler(object):
+
+	"""
+	Example implementation:
+	https://github.com/pyauth/pywarp/blob/master/pywarp/rp.py
+	"""
+
+	def __init__(self, app, webauthn_svc):
+		self.CredentialsService = app.get_service("seacatauth.CredentialsService")
+		self.WebAuthnService = webauthn_svc
+
+		web_app = app.WebContainer.WebApp
+		web_app.router.add_get('/public/webauthn/register-options', self.get_registration_options)
+		web_app.router.add_put('/public/webauthn/register', self.register_credential)
+		web_app.router.add_delete('/public/webauthn/{wacid}', self.remove_credential)
+		web_app.router.add_put('/public/webauthn/{wacid}', self.update_credential)
+		web_app.router.add_get('/public/webauthn', self.list_credentials)
+
+		# Public endpoints
+		web_app_public = app.PublicWebContainer.WebApp
+		web_app_public.router.add_get('/public/webauthn/register-options', self.get_registration_options)
+		web_app_public.router.add_put('/public/webauthn/register', self.register_credential)
+		web_app_public.router.add_delete('/public/webauthn/{wacid}', self.remove_credential)
+		web_app_public.router.add_put('/public/webauthn/{wacid}', self.update_credential)
+		web_app_public.router.add_get('/public/webauthn', self.list_credentials)
```

Comment:
I don't think these should be available on the public API

Reply:
All endpoints needed for the management of my own account need to be accessible from the public API. Auth webui (as it is now) doesn't even have access to the non-public API.

But of course all the endpoints above are protected by login.